app/helpers/application_helper.rb

```diff
@@ -25,4 +25,9 @@ def custom_devise_error_messages(error_msg_hash={})
 
     html.html_safe
   end
+
+
+  def info_site_home_page?
+    params["host"] == ENV['MLN_INFO_SITE_HOSTNAME'] && params["controller"] == "info_site" && params["action"] == "index"
```

Comment:
@daryanypl This is already there in application.yml file. This is existing constant